pages/instructorFileBrowser/instructorFileBrowser.js

```diff
@@ -424,11 +424,7 @@ router.get('/*', function (req, res, next) {
     (err) => {
       if (err) {
         if (err.code === 'ENOENT' && file_browser.paths.branch.length > 1) {
-          res.redirect(
-            `${res.locals.urlPrefix}/${res.locals.navPage}/file_view/${encodePath(
-              file_browser.paths.branch.slice(-2)[0].path
-            )}`
-          );
+          res.redirect(`${req.baseUrl}/${encodePath(file_browser.paths.branch.slice(-2)[0].path)}`);
```

Comment:
The tests involving this code started failing once I made the change to run `git clean` again:

```
invalid input syntax for integer: "file_view"
```

I'm actually really glad they did, because I've been seeing a low volume of these errors on Sentry (e.g. https://sentry.io/organizations/prairielearn-inc/issues/3583272833/?project=6704263) but didn't actually know how to fix them!

Using `req.baseUrl` will be much less prone to failures compared to trying to manually construct the appropriate URL.

Reply:
Nice!